api/managementpb/role.proto

```diff
@@ -0,0 +1,179 @@
+syntax = "proto3";
+
+package management;
+
+option go_package = "api/managementpb;managementpb";
+
+import "protoc-gen-openapiv2/options/annotations.proto";
+import "github.com/mwitkow/go-proto-validators/validator.proto";
+import "google/api/annotations.proto";
+
+message CreateRoleRequest {
+  string title = 1 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  string filter = 2;
```

Comment:
This is a string filter to be applied to metrics. Essentially something like `env=(QA|Staging)`.
Depending on how the UI will be implemented, this will most probably be updated.

Reply:
But this change goes in released API. If we don't have clarity on what we will have on the UI then roles API should be `beta`

Reply:
What is the convention for marking an endpoint `beta` in pmm?

Reply:
https://github.com/percona/pmm/blob/42c60c0915e04f9695de2c6b0d5a0612bbbef690/api/managementpb/ia/alerts.proto#L3-L5

But for that you need to extract definitions in separate folder

Reply:
@michal-kralik what about this?

Reply:
Changed just now